webdriver-spec.html

```diff
@@ -1435,288 +1435,513 @@
+      "<code>&gt;=</code>" operators.
+
+      <p>If the two values do not match, return null.
+
+      <p class=note>Version comparison is left as an implementation detail since
+      each user agent will likely have conflicting methods of encoding the
+      user agent version and standardizing these schemes is beyond the scope of
+      this specification.
+
+      <dt>"<code>platformName</code>"
+      <dd><p>If <var>capability value</var> is not equal to
+      "<code>platformName</code>" entry in <var>matched capabilities</var>,
+      return null.
+
+      <dt>"<code>platformVersion</code>"
+      <dd><p>If <var>capability value</var> is not equal to
+      "<code>platformVersion</code>" entry in <var>matched capabilities</var>,
+      return null.
+
+      <dt>"<code>acceptSslCerts</code>"
+      <dd>
+        <p>If <var>capability value</var> is not a boolean,
+        return <a>error</a> with <a>error code</a> <a>invalid argument</a>.
+
+        <p>If <var>capability value</var> is <code>true</code> and the
+        <a>endpoint node</a> <em>does not</em> support insecure
+        SSL certificates, return null.
+
+        <p>Otherwise, set the "<code>acceptSslCerts</code>" entry in
+        <var>matched capabilities</var> to <var>capability value</var>.
+
+      <dt>"<code>pageLoadStrategy</code>"
+      <dd>
+        <p>If <var>capability value</var> is not a string,
+        return <a>error</a> with <a>error code</a> <a>invalid argument</a>.
+
+        <p>Otherwise, set the "<code>pageLoadStrategy</code>" entry in
+        <var>matched capabilities</var> to <var>capability value</var>.
+
+      <dt>"<code>proxy</code>"
+      <dd>
+        <p>If <var>capability value</var> is not a valid <a>Proxy Configuration</a>,
+        return <a>error</a> with <a>error code</a> <a>invalid argument</a>.
+
+        <p>If the <a>endpoint node</a> does not support the proxy configuration
+        defined in <var>capability value</var>, return null.
+
+        <p>Otherwise, set the "<code>proxy</code>" entry in
+        <var>matched capabilities</var> to <var>capability value</var>.
+
+      <dt>"<code>timeouts</code>"
+      <dd>
+        <p>If <var>capability value</var> is not a valid
+        <a>Session Timeouts Configuration</a> object,
+        return <a>error</a> with <a>error code</a> <a>invalid argument</a>.
+
+        <p>Otherwise, set the "<code>timeouts</code>" entry in
+        <var>matched capabilities</var> to <var>capability value</var>.
+
+      <dt><strong>Otherwise</strong>
+      <dd>
+        <p>If <var>capability name</var> is not recognized by the <a>endpoint node</a>,
+        return null.
+
+        <p>Let <var>processed value</var> be the result of implementation
+        specific steps to match on <var>capability name</var> with
+        <var>capability value</var>.
+
+        <p>If <var>processed value</var> is an error,
+        return <var>processed value</var>.
```

Comment:
Partly because the effect is that the session has not been created, thus making this a more descriptive error to return to the user, partly because existing local end code is written to expect "session not created" to be converted to something like `NoSuchSessionException`, and this is what frameworks catch, and partly because it makes implementing the protocol handshake between OSS and W3C easier.

But mainly because it's more descriptive to the user. The error description should indicate that there was a value that didn't make sense in this particular field.